src/libs/ck-libs/io/ckio.h

```diff
@@ -1,18 +1,25 @@
 #ifndef CK_IO_H
 #define CK_IO_H
 
+#include <vector>
 #include <string>
 #include <pup.h>
 #include <ckcallback.h>
+#include <iostream>
 
 #include "CkIO.decl.h"
 
+
+namespace Ck { namespace IO { class Session; }}
+
+// bool operator==(const Ck::IO::Session& s1, const Ck::IO::Session& s2);
+
 namespace Ck { namespace IO {
   /// Note: The values in options are not currently a stable or working interface.
   /// Users should not set anything in them.
   struct Options {
     Options()
-      : peStripe(0), writeStripe(0), activePEs(-1), basePE(-1), skipPEs(-1)
+      : peStripe(0), writeStripe(0), activePEs(-1), basePE(-1), skipPEs(-1), read_stripe(0), num_readers(0)
```

Comment:
We are mixing camel and snake cases here.

Reply:
Fixed it to all camel case within the Options struct.